src/txdb-leveldb.cpp

```diff
@@ -392,8 +392,7 @@ bool CTxDB::LoadBlockIndex()
                 nLoaded +=10000;
                 if (nLoaded > nHighest) nHighest=nLoaded;
                 if (nHighest < nGrandfather) nHighest=nGrandfather;
-                std::string sBlocksLoaded = ToString(nLoaded) + "/" + ToString(nHighest) + " Blocks Loaded";
-                uiInterface.InitMessage(_(sBlocksLoaded.c_str()));
+                uiInterface.InitMessage(strprintf("%" PRId64 "/%" PRId64 " %s", nLoaded, nHighest, _("Blocks Loaded")));
```

Comment:
bad string format. Some languages require the number between blocks and loaded.

Reply:
Yeah, unfortunately `_(...)` doesn't provide a way to pass arguments for placeholder variants like `tr(...)` does, but it looks like we only use `tr()` in the GUI code. We could expand the `uiInterface` to  support that later, maybe...

Reply:
Ok. Do not worry about it. This is good enough.